mm2src/derives/enum_from/src/lib.rs

```diff
@@ -84,19 +86,96 @@ pub fn enum_from_trait(input: TokenStream) -> TokenStream {
     }
 }
 
+/// `EnumFromStringify` is very useful for generating `From<T>` trait from one enum to another enum
+/// currently, this crate can only convert enum variant with only some basic inner type such as `String`, and `Enum`
+/// type just like the example below. Can not be used for tuple, struct etc for now .
+/// More support will be added.
+///
+///
+/// ### USAGE:
+/// ```rust
+/// use enum_from::EnumFromStringify;
+/// use std::fmt::{Display, Formatter};
+///
+/// // E.G, this converts from whatever Bar is to FooBar::Bar(String) and
+/// // whatever Foo to FooBar::Foo(Foo)
+/// #[derive(Debug, EnumFromStringify, PartialEq, Eq)]
+/// pub enum FooBar {
+///     #[from_stringify("Bar", "String", "Man")]
+///     Bar(String),
+///     #[from_stringify("Foo")]
+///     Foo(Foo),
```

Comment:
Same here. I believe that `EnumFromStringify` should only display the `Foo` into a String and then construct an enum variant.
So the inner type should always be `String`:
```rust
pub enum FooBar {
  #[from_stringify("Bar", "Man")]
  Bar(String),
  #[from_stringify("Foo")]
  Foo(String),   
}
```

Reply:
Not really, one of the aims of the macro is also being able to implement to any type and not just String
```rust
 pub enum FooBar {
  #[from_stringify("Foo")]
  Foo(Foo),   
}

// implement From<Foo> for FooBar
```

Reply:
But this is what `EnumFromInner` does exactly
> // implement From<Foo> for FooBar

Also `#[from_stringify("Foo")]` means that the enum variant is constructed from a **stringified** (from a string representation) `Foo` instance.

Reply:
Ok, do you it's okay to restrict to `String` alone

Reply:
Missed the question. Yes :)